lib/versioner/simple_test.go

```diff
@@ -7,13 +7,14 @@
 package versioner
 
 import (
-	"github.com/syncthing/syncthing/lib/config"
 	"io/ioutil"
 	"math"
 	"path/filepath"
 	"testing"
 	"time"
 
+	"github.com/syncthing/syncthing/lib/config"
```

Comment:
Surprised why this is needed. Nothing new in this file?

Reply:
It's not new. My formatter just moved it down with the rest of the "github.com" imports. I would move it back, but I think it makes since to group these in the same location (even though it is sorting them alphabetically).